test/PHPUnit/Controller/MemoryLeakTest.php

```diff
@@ -40,6 +40,6 @@ public function testMemoryConsumptionNotGrowing($null)
         $this->assertNull($null);
 
         // Test memory consumption is limited to 5 MB for 100 tests
-        $this->assertLessThan(5242880, memory_get_usage(true) - self::$memStart);
+        $this->assertLessThan(5_242_880, memory_get_usage(true) - self::$memStart);
```

Comment:
Since version 7.4, decimal numbers can contain underscores, but do we use them?

Reply:
we use them in other my PRs that were merged by @Ocramius

Reply:
It's really just a style choice. No big effect here, since we're in a test :)

Reply:
@Ocramius 
> It's really just a style choice.

Sure, but not defined, therefore my question. But I have no problem with that, only that we make you a reference and at the same time a standard. 😄

Reply:
I think it's just Rector's preset from @samsonasik: we could provide a CS rule for this in `laminas/laminas-coding-standard`, if we wanted to internalize it.

From the standpoint of this kind of patches, I'd say that it "doesn't matter", as in I prefer constant and gradual improvements, regardless if they introduce some CS inconsistencies that are in a gray zone :)